code-challenges.test.js

```diff
@@ -11,41 +11,110 @@
+//         expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+//     })
+// })
+
+// // const object1 = { number: 15 }
+// // // Expected output: "15 is divisible by three"
+// // const object2 = { number: 0 }
+// // // Expected output: "0 is divisible by three"
+// // const object3 = { number: -7 }
+// // // Expected output: "-7 is not divisible by three"
+
+// // b) Create the function that makes the test pass.
+
+// // Pseudo code:
+// //Input: An object with numbers
+// //Output: An object that decides if the numbers inside are divisible by three
+// //using % iterate through each object's number
+// //using string interpelation, each product returns a string describing the outcome
+
+// const divisibleByThree = (string) => {
+//     let num1 = object1
+//     let num2 = object2
+//     let num3 = object3
+//     if(num1 % 3 === 0) {
+//         retrun `${num1} is divisible by three`
+//     }
+//     if(num2 % 3 === 0) {
+//         return `${num2} is divisible by three`
+//     }
+//     if(num3 %  3 === 0) {
+//         return `${num3} is divisible by three`
+//     }
+// }
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
-
 // a) Create a test with expect statements for each of the variables provided.
-
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+describe('nounsCapetalized', () => {
+    it('takes in an array of words and returns an array with all the words capitalized')
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+    expect(nounsCapetalized(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(nounsCapetalized(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+})
+
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+// // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
\ No newline at end of file
+// Pseudo code:
+//input: an array of words
+//output: an array with all words capetalized
+//using the .toUpperCase we will capetalize all input to receive expected output
+
+const nounsCapetalized = (arr) => {
+    return (arr.charAt(0)).toUpperCase
```

Comment:
Let's think about what this code is doing. You have your parameter `arr`. You are then trying to use `charAt` on the array, which will throw an error because `charAt` can only be used on a string.